integration/tests/epochs/suite.go

```diff
@@ -667,35 +688,34 @@ func (s *Suite) assertNetworkHealthyAfterANChange(ctx context.Context, env templ
 // this func can be used to perform sanity.
 // 1. Ensure sealing continues by comparing latest sealed block from the root snapshot to the current latest sealed block
 func (s *Suite) assertNetworkHealthyAfterVNChange(ctx context.Context, _ templates.Environment, rootSnapshot *inmem.Snapshot, _ *StakedNodeOperationInfo) {
-	bootstrapHead, err := rootSnapshot.Head()
-	require.NoError(s.T(), err)
-
-	header, err := s.client.GetLatestSealedBlockHeader(ctx)
-	require.NoError(s.T(), err)
-
-	// head should now be at-least 20 blocks higher from when we started
-	require.True(s.T(), header.Height-bootstrapHead.Height >= 20, fmt.Sprintf("expected head.Height %d to be higher than head from the snapshot the node was bootstraped with bootstrapHead.Height %d.", header.Height, bootstrapHead.Height))
+	// assert atleast 20 blocks have been finalized since the node replacement
+	s.assertLatestFinalizedBlockHeightHigher(ctx, rootSnapshot, uint64(20))
 }
 
 // assertNetworkHealthyAfterLNChange after an collection node is removed or added to the network
 // this func can be used to perform sanity.
 // 1. Submit transaction to network that will target the newly staked LN by making sure the reference block ID
-// is after the first epoch starts.
-// 2. Ensure sealing continues by comparing latest sealed block from the root snapshot to the current latest sealed block
+// is after the first epoch.
 func (s *Suite) assertNetworkHealthyAfterLNChange(ctx context.Context, _ templates.Environment, rootSnapshot *inmem.Snapshot, _ *StakedNodeOperationInfo) {
-	fullAccountKey := sdk.NewAccountKey().
-		SetPublicKey(unittest.PrivateKeyFixture(crypto.ECDSAP256, crypto.KeyGenSeedMinLenECDSAP256).PublicKey()).
-		SetHashAlgo(sdkcrypto.SHA2_256).
-		SetWeight(sdk.AccountKeyWeightThreshold)
+	// At this point we have reached epoch 1 and our new LN node should be the only LN node in the network.
+	// To validate the LN joined the network successfully and is processing transactions we submit a
+	// create account transaction and assert there are no errors.
+	s.submitSmokeTestTransaction(ctx)
+}
 
+// assertNetworkHealthyAfterSNChange after replacing a consensus node in the test and waiting until
+// the epoch transition we should observe blocks finalizing and we should be able to submit a transaction
+// that will indicate overall network health
+// 1. Ensure sealing continues by comparing latest sealed block from the root snapshot to the current latest sealed block
```

Comment:
I don't see a check for sealing, only finalization.

Reply:
Removed outdated comments https://github.com/onflow/flow-go/pull/1856/commits/151e4aecad9d9365e0c5888625ebcddc303c8448